### ISSUE TITLE
Allow user to select an Archlinux/Manjaro platform from the installer

### ISSUE THREAD

**Issue by tgharib** [manjaro-arm/manjaro-arm-installer-cli#1]
So if you visit this [URL](http://archlinuxarm.org/developers/downloads) you can see all the downloads available for all the platforms.

The plan at the moment is to allow them to select an architecture and the installer will automatically download it, and extract it.

In addition, it would be a good idea to have a list of supported devices that will auto-select the correct architecture since the end user may not know the correct architecture for his/her device.

**Comment by dodgejcr** [comment]
I agree. A simple drop down selector for Device Name instead of architecture. From there possible buttons or another selector for Arch or Manjaro
